src/js/search.js

```diff
@@ -0,0 +1,138 @@
+import axios from 'axios';
+
+const API_KEY = 'eaafeda4857b9c9fecdb45e75f22375a';
+const API_URL = 'https://api.themoviedb.org/3';
+
+const searchMovies = async (query, page) => {
+  const response = await axios.get(
+    `${API_URL}/search/movie?api_key=${API_KEY}&query=${query}&page=${page}`,
+  );
+  return response.json();
+};
+
+const getGenresForSearch = async () => {
+  try {
+    const GET_GENRES_URL = `https://api.themoviedb.org/3/trending/movie/week?api_key=${API_KEY}&page=${page}`;
+    const response = await axios.get(GET_GENRES_URL);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    console.log(error.message);
+  }
+};
+
+const getMoviesForSearch = async (query, page = 1) => {
```

Comment:
ta funkcja to dosłownie to samo co searchMovies, tylko ze tu jest try catch zastosowane